monitoring/ingestion/ingestion-producer-alert.yaml

```diff
@@ -0,0 +1,81 @@
+x-inputs:
+  - name: namespace
+    type: constant
+  - name: job_ingestion_producer
+    type: constant
+  - name: replicas
+    type: constant
+
+groups:
+- name: IngestionProducer
+  rules:
+
+  - alert: IngestionProducerDown
+    Expr: sum(up{namespace="${namespace}", job="${job_ingestion_producer}"}) == 0
+    For:   "30s"
+    Labels:
+      severity: critical
+    Annotations:
+      zenko_service: backbeat-ingestion-producer
+      description: "Ingestion Producer pod has been down for 30 seconds"
+      summary: "Ingestion Producer service is down"
+
+  - alert: IngestionProducerDegraded
+    Expr: sum(up{namespace="${namespace}", job="${job_ingestion_producer}"}) < ${replicas}
```

Comment:
Ingestion producer has replicas "hardcoded" to 1.

Reply:
which is technically not an issue :)

Reply:
But I will change it!

Reply:
Removing this alert because the number of pods is always 1, the first alert should be fine.

Reply:
It is an issue, since `replicas` value is the number of replicas for backbeat processes (expect producer ones) and can be set to a value > 1.
Then, we will fire alerts after  since the sum expression will always return 1.

Reply:
Ah ok..I thought it was per pod.